samples/betaFeatures.js

```diff
@@ -0,0 +1,540 @@
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+
+  //[END speech_transcribe_multilang]
+}
+
+function speechTranscribeMultilangGCS(gcsUri) {
+  // [START speech_transcribe_multilang_gcs]
+
+  const speech = require('@google-cloud/speech').v1p1beta1;
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const uri = path to GCS audio file e.g. `gs:/bucket/audio.wav`;
+
+  const config = {
+    encoding: 'LINEAR16',
+    sampleRateHertz: 44100,
+    languageCode: `en-US`,
+    alternativeLanguageCodes: [`es-ES`, `en-US`],
+  };
+
+  const audio = {
+    uri: gcsUri,
+  };
+
+  const request = {
+    config: config,
+    audio: audio,
+  };
+
+  client
+    .longRunningRecognize(request)
+    .then(data => {
+      const operation = data[0];
+      return operation.promise();
+    })
+    .then(data => {
+      const response = data[0];
+      const transcription = response.results
+        .map(result => result.alternatives[0].transcript)
+        .join('\n');
+      console.log(`Transcription: ${transcription}`);
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END speech_transcribe_multilang_gcs]
+}
+
+function speechTranscribeWordLevelConfidence(fileName) {
+  // [START speech_transcribe_word_level_confidence]
+  const fs = require('fs');
+
+  // Imports the Google Cloud client library
+  const speech = require('@google-cloud/speech').v1p1beta1;
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const fileName = 'Local path to audio file, e.g. /path/to/audio.raw';
+
+  const config = {
+    encoding: `FLAC`,
+    sampleRateHertz: 16000,
+    languageCode: `en-US`,
+    enableWordConfidence: true,
+  };
+
+  const audio = {
+    content: fs.readFileSync(fileName).toString('base64'),
+  };
+
+  const request = {
+    config: config,
+    audio: audio,
+  };
+
+  client
+    .recognize(request)
+    .then(data => {
+      const response = data[0];
+      const transcription = response.results
+        .map(result => result.alternatives[0].transcript)
+        .join('\n');
+      const confidence = response.results
+        .map(result => result.alternatives[0].confidence)
+        .join(`\n`);
+      console.log(
+        ` Transcription: ${transcription} \n Confidence: ${confidence}`
+      );
+
+      console.log(` Word-Level-Confidence:`);
+      const words = response.results.map(result => result.alternatives[0]);
+      console.log(`WORDS: `);
+      words.forEach(a => console.log(JSON.stringify(a, null, 4)));
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END speech_transcribe_word_level_confidence]
+}
+
+function speechTranscribeWordLevelConfidenceGCS(gcsUri) {
+  // [START speech_transcribe_word_level_confidence_gcs]
```

Comment:
Is a GCS version required? I can't find this region tag in [python-docs-samples's `beta_features.py`](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/speech/cloud-client/beta_snippets.py)

Reply:
----

Meh, **nevermind**, I see the GCS versions in the Java samples ([Recognize.java](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/speech/cloud-client/src/main/java/com/example/speech/Recognize.java))